hooks/hook_frontpage.php

```diff
@@ -43,7 +43,7 @@ function oidc_hook_frontpage(&$links)
     }
 
     $links['federation']['oidcregistry'] = [
-        'href' => \SimpleSAML\Module::getModuleURL('oidc/clients/'),
+        'href' => \SimpleSAML\Module::getModuleURL('oidc/admin-clients/'),
```

Comment:
There is also hooks/hook_federationpage.php (used when usenewui option is enabled).

Reply:
fixed